tests/framework/results/test_stratification.py

```diff
@@ -140,3 +141,18 @@ def test_stratification_call_raises(
     my_stratification = Stratification(name, sources, categories, mapper, is_vectorized)
     with pytest.raises(expected_exception):
         raise my_stratification(STUDENT_TABLE)
+
+
+def test_results_manager_setup(mocker):
```

Comment:
Isn't this test identical to the one above in `test_manager`?

Reply:
Ah yeah I forgot to clean this one up.